### PR TITLE
fix(ColorPicker): explicitly provide default props to Checkboard component

### PR DESCRIPTION
**Pull request by aliameur**
### Title:
Fix `react-color` Checkboard Component's Outdated `defaultProps`

### Description:

Default props in functional components were [removed](https://react.dev/blog/2024/04/25/react-19-upgrade-guide#removed-proptypes-and-defaultprops) in React 19. This PR adjusts the `Checkboard` component’s usage to prevent issues arising from this change.

Solves #76.

### Changes

- Provides explicit `size`, `white`, and `grey` props to specify the `Checkboard` component's appearance.
- Adds type casting to `unknown` for compatibility.

### CODE REVIEW

**Review by maclong9 — COMMENTED** [review]
Looks good.

**Review by igorpimentel23 — APPROVED** [review]

**Comment by SologodTW** [comment]
Is this fix being released soon?

**Comment by aliameur** [comment]
Merging requires approving review from a reviewer with write access. I can't do much atm. If you need the change now I recommend patching the package yourself with the same changes @SologodTW

**Review by thebiggianthead — APPROVED** [review]

**Pull request merged**
